api/webscraper/utils/scraper_utils.py

```diff
@@ -0,0 +1,30 @@
+import requests
+import json
+import pandas as pd
+from io import BytesIO
+import urllib
+from dotenv import load_dotenv
+import os
+
+load_dotenv('.env.local')
+
+google_maps_api_key = os.environ.get('NEXT_PUBLIC_GOOGLE_MAPS_API_KEY')
+
+def check_status(status):
+    if status is None:
```

Comment:
In this case, let's return NULL for now.

Reply:
Hi Monique! just wanted to clarify--in python, `None `is a special keyword used to refer to null objects. So do you want me to return a string `"NULL"`?

Reply:
return `None` then ! But make sure when you write it in the database it will be NULL.